api/handlers/configure.go

```diff
@@ -379,5 +379,7 @@ func removeBridgeFromIngress(ingress *networking.Ingress) {
 }
 
 func getHostForBridge(keptnDomain string) string {
-	return "bridge.keptn." + keptnDomain
+	// check if the domain contains a port. If yes, only the first part without the port will be used
+	split := strings.Split(keptnDomain, ":")
```

Comment:
As far as I know, we have to differentiate between VirtualServices and Ingress:
- VirtualServices allow Ports
- Ingress does not allow Ports

Reply:
Thank you for clarifying that also VirtualServices do not allow ports.